@here/harp-mapview/lib/text/TextElementState.ts

```diff
@@ -225,6 +244,10 @@ export class TextElementState {
         }
 
         this.m_textRenderState = new RenderState();
+        this.m_textLayoutState = new LayoutState();
```

Comment:
Do we really need to initialize always the layout state? Can we avoid this initialization for labels that are placed in its default layout position and only initialize the layout state when they're moved to an alternative position?

Reply:
Again let's go back to previous comments.